draft-ietf-rats-reference-interaction-models.md

```diff
@@ -160,11 +160,15 @@ A PKIX Certificate is an X.509v3 certificate as specified by {{RFC5280}}.
 In the context of this document, the term "Remote" does not necessarily refer to a remote entity in the scope of network topologies or the Internet.
 It rather refers to decoupled systems or entities that exchange the Conceptual Message type called Evidence {{-RATS}}.
 This conveyance can also be "Local", if the Verifier role is part of the same entity as the Attester role, e.g., separate system components of the same Composite Device (a single RATS entity), or the Verifier and Relying Party roles are hosted by the same entity, for example in a cryptographic key broker system.
-Even if an entity takes on two or more different roles, the functions they provide typically reside in isolated environments that are components of the same entity. Examples of such isolated environments include a Trusted Execution Environment (TEE), Baseboard Management Controllers (BMCs), as well as other physical or logical protected/isolated/shielded Computing Environments (e.g., embedded Secure Elements (eSE) or Trusted Platform Modules (TPM)). Readers of this document should be familiar with the concept of Layered Attestation as described in {{Section 3.1 of {{-RATS}} and the definition of Attestation as described in {{-RIV}}.
+Even if an entity takes on two or more different roles, the functions they provide typically reside in isolated environments that are components of the same entity. Examples of such isolated environments include a Trusted Execution Environment (TEE), Baseboard Management Controllers (BMCs), as well as other physical or logical protected/isolated/shielded Computing Environments (e.g., embedded Secure Elements (eSE) or Trusted Platform Modules (TPM)). It is useful but not necessary for readers of this document to be familiar with the Conceptual Data/Messages flows as described in {{Section 3 of {{-RATS}} and the definition of Attestation in general as described in {{-RIV}}.
 
 # Scope and Intent
 
-This document outlines the interaction models between Attesters and Verifiers to convey Evidence.
+This document outlines three very common interaction models between RATS roles.
+In this document the interaction models illustrated focus on the conveyance of Evidence about boot-time integrity from Attesters to Verifiers.
+In the scope of Evidence conveyance not every detail is covered in this document.
+Details around Evidence about run-time integrity, for example, or not explicitly highlighted, but the included model descriptions still function as a basis to build corresponding model extensions on.
+While the three interaction models (and their sub-variants) are not an exhaustive list of all potential models, they do cover a vast majority of conveyance models for Conceptual Messages implemented in the Internet.
```

Comment:
This more nuanced description seems at odds with L146 which claims "total and unconfined coverage" :-)

Reply:
The original L146 states:

> _"[t]he methods described in this document can also be applied to the conveyance of ${CONCEPTUAL_MESSAGE}"_

which, at least to this reader, sounds like there are no limits to the potential coverage of the described methods.

The amended version of L146 that you propose [in the comment above](https://github.com/ietf-rats-wg/draft-ietf-rats-reference-interaction-models/pull/56/files#r1799822298) makes it more in line with L171.

Reply:
So, you are okay with the L146 as per this PR and this comment can be resolved? I mean, the intent was to address your comment, after all.